Mh/play playable/combat rules

bleehu opened the pull request:
A mostly-complete reorganization of the combat rules, which were a little out-of-order and cloogy. No rules were intentionally changed, but some summary prose sections were added. The intention was to make our combat rules easier to understand on first read, as well as easier to skim.

bleehu committed:
Broke Combat Rules up into more readable and searchable documents.
Also rearranged the combat rules to group similar rules together
and re-worded some of the more confusing rules.
 Changes to be committed:
	modified:   ../01_01_Basic_Rules.txt
	deleted:    ../01_02_Combat_Rules.txt
	new file:   1_Combat_Rules.txt
	new file:   2_Shooting.txt
	new file:   3_Melee_Combat.txt
	new file:   4_Armor_Rules.txt
	new file:   5_Clearing_Rooms.txt

bleehu committed:
Broke Combat Rules up into more readable and searchable documents.
Also rearranged the combat rules to group similar rules together
and re-worded some of the more confusing rules.
 Changes to be committed:
	modified:   ../01_01_Basic_Rules.txt
	deleted:    ../01_02_Combat_Rules.txt
	new file:   1_Combat_Rules.txt
	new file:   2_Shooting.txt
	new file:   3_Melee_Combat.txt
	new file:   4_Armor_Rules.txt
	new file:   5_Clearing_Rooms.txt

some artifacts may occur, attempted messy interactive rebase
with --allow-empty

 You are currently rebasing branch 'dev' on '3adf6af'.

Turtlelord26 commented on 01_01_Basic_Rules.txt:
```diff
@@ -390,6 +390,8 @@ Some class feats and other abilities can increase the number of Nanites you can
 combat at a rate of 3 Nanites per turn. 
 
+=====  =====
+
```
Found a git artifact.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+
+	Combat starts when either you or an enemy decides they would like to make an 
+attack (or as the GM decrees). During combat, the flow of the game changes so 
+that both player characters and the Non-Player Characters which are run by the
```
Gr: "Non-Player characters *that are run by..."

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+non-combat. When the encounter is time-sensitive, or if the movement of the 
+sneaker is relative to the movment of what else is happening, then stealth can
+require the same turn-based flow as combat. When the sneaking is happening under
+less pressure, it can often skip the rigor of turn-based gameplay.
```
A good addition.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+	If a character succeeds on their stealth skill checks, gets the drop on
+another character, and decides to attack, then the character or characters that 
+were hidden get to take a whole round without the surprised characters getting
+a turn. This is called the Surprise Round.
```
May need another sentence to ensure maximum clarity. Technically an added rule here, but it's pretty well tested.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+and Held Actions, below. After the Reaction occurs, play continues as normal. 
+If it was an opponent's turn, they continue their turn. 
+
+	A character may use up to three Reaction per round, regardless of how 
```
Not your typo, but Sp: Reactions

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+
+Talking: Free Action
+Performing a Skill Check (other than Getting Bearings): 1 Action
+Spotting(PER) checks to look for enemies: Free Action
```
I was under the impression Awareness to notice enemies was free, but Spotting to seek out enemies, such as from underbrush, was an Action.

bleehu replied:
I think you're right; I'll change it.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+Percentiles under their armor in order to see if their armor blocked the bullet, 
+and from there, how much damage they receive. If the defending player has no 
+armor, they armor roll for a target 50 Percentile. If the armor roll is 50% or 
+above, they receive half-damage. Below 50%, full damage.
```
Out of scope, iirc we're killing this half-damage rule.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+	When players take more than 70 damage, they must roll a Shock save at a DC 
+of the damage taken. On fail they pass out. on success they stay conscious. When 
+players fail, they may roll each turn at a DC of 70 to wake up. They continue to 
+roll either for (15 - FORT) turns, or until they pass.
```
Out of scope, iirc we're killing the massive damage Shock rule too.

Turtlelord26 commented on Combat Rules/1_Combat_Rules.txt:
```diff
@@ -0,0 +1,392 @@
+*Base Damage is typically the damage of the weapon without any other character 
+bonuses, but Base Damage can be increased by certain classes and abilities.
+
+**Increases to 200 max damage at level 10.
```
We seem to be missing some lines between 386 and 388.

> 	Ranged weapon stealth-attacks (including explosive grenades but not any 
other thrown weapons) have -1 miss chance to hit if 'adequate' aiming time is 
given (DM's discretion: how rushed you are, or how skilled you are at stealth, 
etc). Ranged weapon stealth-attacks gain +20 bonus damage.
>
>	If you stealth-attack a target in melee, your target has a Guard of 0. As
normal (see section, "Melee Combat"), if you roll 3+ above the opponent's Guard, 
you ignore the opponent's armor and deal full damage. This signifies that the 
target gets hit in whatever you deem to be the most vulnerable; i.e., the head, 
neck, or the chest (over the heart, etc). 

Edit: Ah, I found them in the other files. I would argue they should be kept in the rules subsection that references starting a fight, since they're less likely to occur in the middle. Otherwise, the footnotes should be moved to refer to the relevant paragraphs or terminated.

Edit again: On balance, I like terminating the footnotes, they seem vestigial. In the end I suspect all the stealth stuff will move to a separate Stealth Combat subsection when the Tech Stealth project rolls around, so things are probably fine as they are.

Turtlelord26 reviewed (approved):
This is beautiful and I appreciate your willingness to have to update CXDocs to keep up with page creep.

bleehu committed:
added Combat Rules Section
 Changes to be committed:
	modified:   ../01_01_Basic_Rules.txt
	modified:   1_Combat_Rules.txt
